app/services/post_owner_changer.rb

```diff
@@ -23,6 +24,13 @@ def change_owner!
       post.set_owner(@new_owner, @acting_user, @skip_revision)
       PostAction.remove_act(@new_owner, post, PostActionType.types[:like])
 
+      if post.post_number == 1
+        TopicUser.change(old_owner.id, @topic.id, notification_level: 2) if old_owner
```

Comment:
I'm not sure if we should change the `notification_level` of the previous owner. Let's discuss this on meta.
If we change it, should we set it to the user's notification level for that category or group or always to `tracking`?